hypershift-operator/controllers/nodepool/haproxy.go

```diff
@@ -7,6 +7,7 @@ import (
 	"fmt"
 	"html/template"
 	"net/url"
+	"slices"
```

Comment:
Konflux bot is right, fails to compile
```
CGO_ENABLED=1 GO111MODULE=on GOFLAGS=-mod=vendor go build -gcflags=all='-N -l' -o bin/hypershift-operator ./hypershift-operator
hypershift-operator/controllers/nodepool/haproxy.go:10:2: package slices is not in GOROOT (/usr/lib/golang/src/slices)
```

Reply:
I see, I'll update that logic to not use slices. We are covered by the units.